### PR TITLE
eFMer.BoincTasks version 1.78

### PR DESCRIPTION
**Pull request by Toby-Broom**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36921)

### CODE REVIEW

File: manifests/e/eFMer/BoincTasks/1.7.8/eFMer.BoincTasks.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+PackageIdentifier: eFMer.BoincTasks
+PackageVersion: 1.7.8
+PackageLocale: en-US
+Publisher: eFMer
+PublisherUrl: https://efmer.com/
+PublisherSupportUrl: https://forum.efmer.com/
+PrivacyUrl: https://efmer.com/
+Author: Fred Melgert
+PackageName: BoincTasks
```
Comment (OfficialEsco) [inline manifests/e/eFMer/BoincTasks/1.7.8/eFMer.BoincTasks.locale.en-US.yaml:12]:
```suggestion
PackageName: Boinc Tasks
```

File: manifests/e/eFMer/BoincTasks/1.7.8/eFMer.BoincTasks.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: eFMer.BoincTasks
+PackageVersion: 1.7.8
```
Comment (OfficialEsco) [inline manifests/e/eFMer/BoincTasks/1.7.8/eFMer.BoincTasks.installer.yaml:5]:
```suggestion
PackageVersion: 1.78
```
This needs to match ARP

File: manifests/e/eFMer/BoincTasks/1.7.8/eFMer.BoincTasks.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: eFMer.BoincTasks
+PackageVersion: 1.7.8
```
Comment (OfficialEsco) [inline manifests/e/eFMer/BoincTasks/1.7.8/eFMer.BoincTasks.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.78
```
This needs to match ARP

File: manifests/e/eFMer/BoincTasks/1.7.8/eFMer.BoincTasks.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: eFMer.BoincTasks
+PackageVersion: 1.7.8
```
Comment (OfficialEsco) [inline manifests/e/eFMer/BoincTasks/1.7.8/eFMer.BoincTasks.yaml:5]:
```suggestion
PackageVersion: 1.78
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image

**Comment by OfficialEsco** [comment]
You also need to change the version folder name from `manifests/e/eFMer/BoincTasks/1.7.8/` to `manifests/e/eFMer/BoincTasks/1.78/`

**Comment by Toby-Broom** [comment]
@OfficialEsco Can I change it here? or is it better to resubmit?

**Comment by OfficialEsco** [comment]
Press . to open online VSCode

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**